apps/cookbook/src/app/showcase/slides-showcase/slides-showcase.component.html

```diff
@@ -36,14 +33,23 @@ <h2>Inherit height</h2>
   to each slide to make them equal height as seen in the example below.
 </p>
 
-<cookbook-iphone
-  viewMode="full-size"
-  [showViewModeToggle]="true"
-  src="/examples/slides-height/"
-  showExternalLink="true"
-></cookbook-iphone>
+<cookbook-example-viewer [html]="heightExample.template">
+  <div class="example-frame">
+    <cookbook-slides-height-example #heightExample></cookbook-slides-height-example>
+  </div>
+</cookbook-example-viewer>
+
+<h2>Custom heading</h2>
```

Comment:
We could change the heading to be more informative, and maybe add a line about how this makes it possible for slide titles to participate in the heading structure of the page, and thereby also show up in screen reader software etc.

```suggestion
<h2>Heading Level</h2>
```